code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -35,6 +35,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
 91-100: Dangerously toxic - swift death
 */
 
+/datum/reagent/consumable/ethanol/New()
+	addiction_types = list(/datum/addiction/alcohol = 0.05 * boozepwr)
```

Comment:
Would this mean that the Crevice Spike and Fanciulli actually reduce your addiction to alcohol every tick?

Also, that puts this at a range of -0.5 to 15 a tick, if you want to add a comment here.